src/injected.js

```diff
@@ -66,13 +66,16 @@ ghImport('jquery').then(function($) {
 		}
 	
 		var createListeners = function() {
-			// This handles partial updates
+			

```

Comment:
You seem to like empty lines... can you do me a favor and scale back on them? My mouse wheel needs a break... ;)